updater/Gemfile.lock

```diff
@@ -322,4 +322,4 @@ DEPENDENCIES
   webmock (~> 3.18)
 
 BUNDLED WITH
-   2.4.11
+   2.3.26
```

Comment:
This looks off?

* https://github.com/dependabot/dependabot-core/pull/7046

Even though you ran this from a `2.3.26` `bundler` version, I suspect it'll be okay if you simply manually put it back to `2.4.11` w/o having to re-run with the newer `bundler` version... 🙉 🙈

Reply:
Good catch :)